extensions/objects/templates/filepath_editor.html

```diff
@@ -17,17 +17,17 @@
     <div ng-repeat="filepath in filepaths track by $index">
       <label>
         <input type="radio" ng-model="localValue.label"
-               name="choiceGroup" id="<[filepath]>"
+               name="choiceGroup" id="simbly"
```

Comment:
Yeah i added that when i was experimenting with jQuery. It's not needed anymore.